ui/src/pages/Login.tsx

```diff
@@ -6,45 +6,91 @@ import * as React from "react";
 import { Redirect } from "react-router-dom";
 import styled from "styled-components";
 
+import ShoeLogo from "../assets/images/shoeproject-logo.svg";
+import PeopleWalking from "../assets/images/walking-people.svg";
 import { FailureState, useAuth } from "../hooks/auth";
-import { LoginMessageText, LoginTitleText } from "../styles/typography";
+import { colors } from "../styles/index";
+import {
+  LoginHeader,
+  LoginMessageText,
+  LoginTitleText,
+} from "../styles/typography";
+
+const PeopleFooter = styled.img`
+  z-index: 1;
```

Comment:
I genuinely can't get the illustration to move in front of the material ui card 😭  I'll try styling the card with `createStyles` and see if the z-index registers properly.
![image](https://user-images.githubusercontent.com/19617248/115948402-2b1b6080-a49c-11eb-82d0-4618d5d72c79.png)